tests/integration/api/test_workbenches.py

```diff
@@ -38,3 +41,29 @@ def test_vulnerability_output(self, client):
             assert False, u'TenableIOApiException should have been thrown for bad ID.'
         except TenableIOApiException as e:
             assert e.code is TenableIOErrorCode.BAD_REQUEST, u'Appropriate exception thrown.'
+
+    def test_export(self, app, client):
+        file_id = client.workbenches_api.export_request(
+            WorkbenchesApi.FORMAT_NESSUS,
+            WorkbenchesApi.REPORT_VULNERABILITIES,
+            WorkbenchesApi.CHAPTER_VULN_BY_ASSET,
+        )
+        assert file_id, u'The `export_request` method returns a valid file ID.'
+
+        export_status = self.wait_until(lambda: client.workbenches_api.export_status(file_id),
+                                        lambda status: status == WorkbenchesApi.STATUS_EXPORT_READY)
+
+        assert export_status == WorkbenchesApi.STATUS_EXPORT_READY, u'Workbench export is ready.'
+
+        iter_content = client.workbenches_api.export_download(file_id, False, None)
```

Comment:
I'm getting an error when running this test if I include the None as the third param, it passes if I remove this param.

```
============================================================================================= FAILURES =============================================================================================
__________________________________________________________________________________ TestWorkbenchesApi.test_export __________________________________________________________________________________

self = <tests.integration.api.test_workbenches.TestWorkbenchesApi object at 0x1059dfd68>, app = <tests.conftest.App object at 0x1059dfe80>
client = <tenable_io.client.TenableIOClient object at 0x1059fb6a0>

    def test_export(self, app, client):
        file_id = client.workbenches_api.export_request(
            WorkbenchesApi.FORMAT_NESSUS,
            WorkbenchesApi.REPORT_VULNERABILITIES,
            WorkbenchesApi.CHAPTER_VULN_BY_ASSET,
        )
        assert file_id, u'The `export_request` method returns a valid file ID.'
    
        export_status = self.wait_until(lambda: client.workbenches_api.export_status(file_id),
                                        lambda status: status == WorkbenchesApi.STATUS_EXPORT_READY)
    
        assert export_status == WorkbenchesApi.STATUS_EXPORT_READY, u'Workbench export is ready.'
    
        iter_content = client.workbenches_api.export_download(file_id, False, None)
        download_path = app.session_file_output('test_workbench_export')
    
        assert not os.path.isfile(download_path), u'Workbench report does not yet exist.'
    
        with open(download_path, 'wb') as fd:
>           for chunk in iter_content:

tests/integration/api/test_workbenches.py:64: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

string = b'<?xml version="1.0" ?>\n<NessusClientData_v2>\n<Report name="workbenchexport" xmlns:cm="http://www.nessus.org/cm"></ReportHost></Report></NessusClientData_v2>', slice_length = None

    def iter_slices(string, slice_length):
        """Iterate over slices of a string."""
        pos = 0
        while pos < len(string):
>           yield string[pos:pos + slice_length]
E           TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'

/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/requests/utils.py:350: TypeError
```